en/flight_modes_rover/differential.md

```diff
@@ -0,0 +1,132 @@
+# Drive Modes (Differential Rover)
+
+Flight modes (or more accurately "Drive modes" for ground vehicles) provide autopilot support to make it easier to manually drive the vehicle or to execute autonomous missions.
+
+This section outlines all supported drive modes for differential rovers.
+
+For information on mapping RC control switches to specific modes see: [Basic Configuration > Flight Modes](../config/flight_mode.md).
+
+::: warning
+Selecting any other mode than those listed below will either stop the rover or can lead to undefined behaviour.
+:::
+
+## Manual Modes
+
+Manual modes require stick inputs from the user to drive the vehicle.
+
+![Manual Controls](../../assets/airframes/rover/flight_modes/manual_controls_differential_rover.png)
```

Comment:
You may have answered this elsewhere, but not here. Are the stick behaviours hard coded? I mean using a taranis you might have a mode 0 or mode 1 setup, and then right could be left and so on.

Is this some kind of standard setup, and if so, how is it named for Rover?